### DISCUSSION TITLE
422 (Unprocessable Entity)

### DISCUSSION THREAD

**Discussion by azri1521** [symfony/symfony#47347] [Q&A] [answered]
Hi,

I got this error message 422 (Unprocessable Entity) with symfony 6.1.2 version. it happen when submit the form with validation error. when the form post back it show 422 (Unprocessable Entity)  message. I though my mistake so I create new symfony 6.1.3 project with simple database and form. 

database only got 1 table with field 'label'.
fund form texttype with label NotBlank validation.

using CRUD create controller and default CRUD templates.

when submit console return
POST http://localhost:8002/fund/new 422 (Unprocessable Entity)
Navigated to http://localhost:8002/fund/new

server running using symfony server:start using php 8.1

if using cloud server php 8.1 symfony 6.1.3 same as local server got this message
POST http://symfony613.atasawan.my/fund/new 422 (Unprocessable Content)

try to google but did not find solution.

Regards

**Comment by MatTheCat** [comment, marked as answer]
Since MakerBundle 1.33 and Symfony 5.3 a new `renderForm` method is used, which returns a 422 response code when a submitted form is invalid: https://github.com/symfony/symfony/blob/a57252fb11b4cd856476aa4e4dd359404d3537fb/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L270-L298

What problem does this cause to you?

**Reply by azri1521** [reply]
no problem. I thought maybe I miss something in version 6 setting. 

Thank you

**Comment by fodger** [comment]
Hello,
this new behavior is a mistake with external partners who can send sometimes some POST request, such as etp partners with post payment requests, they receive a 422 error only because it's not a symfony form, it is a no sens.

**Reply by stof** [reply]
if there is no Symfony form, it won't return a 422. It will return a 422 only when you have a submitted Symfony form with errors.

**Reply by fodger** [reply]
Are you really sure ? 
It occurs since I updated SF with latest 5.4 version, how to explain that SF gives a 422 code after receiving a classical POST request from a partner ? Before updating there was no http 422 code.

**Reply by stof** [reply]
I suggest you to open a new support request providing details about your issue instead of commenting on this discussion that is already marked as answered. My guess is that your problem might not be the same than in the original discussion here (which was not even a problem).
